OmniDB/OmniDB_app/static/OmniDB_app/js/passwords.js

```diff
@@ -112,3 +112,45 @@ function checkPasswordPrompt(p_database_index, p_callback_function, p_cancel_cal
     );
   }
 }
+
+
+function showMasterPassPrompt(p_message) {
```

Comment:
This modal should look like any other password set/change prompt. Please add the password confirmation field and add the necessary validations to ensure that both passwords match.
The modal should not be submittable until form is valid.